src/main/java/lotto/application/service/LottoService.java

```diff
@@ -0,0 +1,47 @@
+package lotto.application.service;
+
+import lotto.domain.generator.LottoNumberGenerator;
+import lotto.domain.model.*;
+import lotto.domain.result.LottoResult;
+import lotto.domain.result.ResultCalculator;
+import lotto.infrastructure.repository.LottoRepository;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class LottoService {
+    private static final int LOTTO_TICKET_PRICE = 1000;
+
+    private final LottoNumberGenerator numberGenerator;
+    private final LottoRepository lottoRepository;
+
+    public LottoService(LottoNumberGenerator numberGenerator, LottoRepository lottoRepository) {
+        this.numberGenerator = numberGenerator;
+        this.lottoRepository = lottoRepository;
+    }
+    public void purchaseLotto(int amount) {
```

Comment:
```suggestion
    }
    
    public void purchaseLotto(int amount) {
```

메소드간 줄간격 컨벤션 지키면 좋을 것 같습니다

Reply:
감사합니다!